posixlib/src/main/scala/scala/scalanative/posix/fcntl.scala

```diff
@@ -15,39 +15,6 @@ object fcntl {
 
   def fcntl(fd: CInt, cmd: CInt, flags: CInt): CInt = extern
```

Comment:
Because I have been kicked by this file before, I _did_ check it against the posix specification.
I found it to be problematic, from before this PR.  I believe that an Issue should be created
and this file, in its new location, marked for scrutiny, probably by at least two people.

This file is known to be incomplete, especially the constants. I did not try to figure out the missing values.
I did, however, look at the example for fcntl().  By my reading, that can not be implemented in SN with
the current method declarations, or at least not cleanly.  fcntl() and, I believe, open() are defined in
posix as having a variable number of arguments.  That is difficult in SN.  If it is too difficult, then
the methods should be marked as non-posix but passably useful in the file.

Reply:
Maybe add an issue for this as I do not know.